src/impls.rs

```diff
@@ -93,7 +93,7 @@ impl Format for f32 {
 impl Format for Str {
     fn format(&self, fmt: &mut Formatter) {
         if fmt.needs_tag() {
-            let t = internp!("{:str}");
+            let t = internp!("{:istr}");
```

Comment:
we forgot to update this format string when we renamed `{:str}` to `{:istr}` some time ago